api/bertymessenger.proto

```diff
@@ -207,8 +207,8 @@ message AppMessage {
     TypeUserMessage = 1;
     TypeUserReaction = 2;
     TypeGroupInvitation = 3;
-    TypeSetGroupName = 4;
-    TypeSetUserName = 5;
+    TypeSetGroupInfo = 4;
+    TypeSetUserInfo = 5;
```

Comment:
Ahah, I don't know if it was expected or if you were aware, but since you reuse the same number for those calls and since those calls still have the "name" as a string and number 1, we're backward-compatible on this part (if we re-read an old database, the marshaled protobuf message of `SetGroupName{Name: "foobar"}` will automatically be unmarshalled as `SetGroupInfo{DisplayName: "foobar"}` :)

Reply:
unexpected ^^